distod/src/main/scala/com/github/codelionx/distod/actors/worker/Worker.scala

```diff
@@ -44,20 +41,24 @@ class Worker(workerContext: WorkerContext) extends CandidateGeneration {
   import Worker._
   import workerContext._
 
+
   def start(): Behavior[Command] = initialize()
 
-  def initialize(): Behavior[Command] = {
+  private def initialize(): Behavior[Command] = {
     partitionManager ! LookupAttributes(partitionEventMapper)
 
     Behaviors.receiveMessagePartial {
       case WrappedPartitionEvent(AttributesFound(attributes)) =>
         context.log.trace("Worker ready to process candidates: Master={}, Attributes={}", master, attributes)
         master ! DispatchWork(context.self)
         behavior(attributes)
+
+      case Stop =>
+        Behaviors.stopped
     }
   }
 
-  def behavior(attributes: Seq[Int]): Behavior[Command] = Behaviors.receiveMessage {
+  private def behavior(attributes: Seq[Int], stopped: Boolean = false): Behavior[Command] = Behaviors.receiveMessage {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=4969716)